config/missing-page-redirector.php

```diff
@@ -7,7 +7,14 @@
      * `Spatie\MissingPageRedirector\Redirector\Redirector`-interface
      */
     'redirector' => \Spatie\MissingPageRedirector\Redirector\ConfigurationRedirector::class,
-
+    
+    /*
+     * This allows you to trigger the redirects on all kind of HTTP codes.
+     * It also it also accepts an array.
+     * To trigger redirects on any HTTP code leave it empty
+     */
+    'status_code' => \Symfony\Component\HttpFoundation\Response::HTTP_NOT_FOUND,
```

Comment:
Let's make this an array by default so we can lose that `It also it also accepts an array.` comment

If it's an empty array then package should redirect no matter what.